BackEnd/scripts/generateTeamModel.ts

```diff
@@ -61,15 +61,16 @@ async function buildTeam(seasonId: number, displayName: string, abbreviation: st
   await SaveObjects(rows, TeamModel);
 }
 
-async function buildPlayersInTeams(seasonId: number, teamId: number, playersNames: string[]) {
+async function buildPlayersInTeams(seasonId: number, teamId: number, riotNames: string[]) {
   await ensureConnection();
 
   let playerPuuids: string[] = [];
 
-  for (let player of playersNames) {
+  for (let player of riotNames) {
```

Comment:
If we are going to rename anyway, can this be more descriptive? Like are tags included?

Reply:
Maybe `nameWithTag`?

Reply:
This is a script so i didnt really think too much about it

Reply:
Oh lol mb. Didn't notice it was a scripts file.